openmp/libomptarget/plugins-nextgen/amdgpu/src/rtl.cpp

```diff
@@ -3122,6 +3180,12 @@ struct AMDGPUPluginTy final : public GenericPluginTy {
 
   /// The device representing all HSA host agents.
   AMDHostDeviceTy *HostDevice;
+
+  /// Is the plugin associated with an APU?
+  bool IsAPU;
+
+  /// Value of HSA_XNACK environment variable.
+  BoolEnvar OMPX_HSAXnack;
```

Comment:
Why is this `OMPX`? I think those names are sort-of reserved for OMP extensions, right?

Reply:
I think we use things like `LIBOMPTARGET_AMDGPU_TEAMS_PER_CU`, but this is really piggy backing on the HSA env var. We will later need XXXXEnvar with multiple names, for now, OMPX_HSAXnack seems ok to me.

Reply:
I changed the name to HSAXnackEnv because it does not refer to an OpenMP extension.